packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -107,8 +108,12 @@
     "@aws-cdk/core": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
     "@aws-cdk/region-info": "0.0.0",
-    "constructs": "^3.3.69"
+    "constructs": "^3.3.69",
+    "object-hash": "^2.2.0"
   },
+  "bundledDependencies": [
+    "object-hash"
+  ],
```

Comment:
You should be able to do crypto without having to use this dependency.

Nodejs' crypto library should suffice - https://nodejs.org/api/crypto.html#cryptocreatecipherivalgorithm-key-iv-options